### ISSUE TITLE
[DocDB][Read Committed] LookupByIdRpc(tablet: 0fed15a16c2d4f70b4bcd5cd52c0852f, num_attempts: 1) failed: Tablet deleted:

### ISSUE THREAD

**Issue by shishir2001-yb** [yugabyte/yugabyte-db#18891]
Jira Link: [DB-7752](https://yugabyte.atlassian.net/browse/DB-7752)
### Description

Tried on version **2.19.3.0-b27**
The following error was thrown in a transaction with read committed isolation level while inserting rows in a batch 

`
[Not found (yb/master/catalog_manager.cc:11692): LookupByIdRpc(tablet: 0fed15a16c2d4f70b4bcd5cd52c0852f, num_attempts: 1) failed: Tablet deleted: Table deleted at 2023-08-29 09:11:05 UTC (split child tablet IDs []),]
`

Tserver and Master flags:
```
"ysql_output_buffer_size": "26214400",
"enable_wait_queues": "true",
"enable_deadlock_detection": "true",
"yb_enable_read_committed_isolation": "true"
```

Logs
[172.151.23.235-logs.tar.gz](https://github.com/yugabyte/yugabyte-db/files/12463595/172.151.23.235-logs.tar.gz)
[172.151.47.184-logs.tar.gz](https://github.com/yugabyte/yugabyte-db/files/12463599/172.151.47.184-logs.tar.gz)
[172.151.49.111-logs.tar.gz](https://github.com/yugabyte/yugabyte-db/files/12463601/172.151.49.111-logs.tar.gz)

### Warning: Please confirm that this issue does not contain any sensitive information

- [X] I confirm this issue does not contain any sensitive information.

[DB-7752]: https://yugabyte.atlassian.net/browse/DB-7752?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

**Comment by shishir2001-yb** [comment]
Find the insert query below
`
Caused by: java.sql.BatchUpdateException: Batch entry 0 INSERT INTO tbl_no_explicit_lock(k, v1, v2, v3, v4, v5, v6, v7, v8, v9, v10, v11, v12, v13, v14, v15, v16, v17, v18, v19, v20, v21, v22, v23, v24) VALUES ( 'be8ef4cb-ec72-4679-a488-5af1eb848d36:1', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:1', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:1', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:1', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:1', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:1', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:1', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:1', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:1', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:1', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:1', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:1', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:1', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:1', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:1', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:1', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:1', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:1', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:1', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:1', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:1', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:1', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:1', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:1', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:1'),( 'be8ef4cb-ec72-4679-a488-5af1eb848d36:2', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:2', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:2', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:2', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:2', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:2', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:2', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:2', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:2', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:2', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:2', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:2', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:2', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:2', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:2', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:2', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:2', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:2', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:2', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:2', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:2', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:2', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:2', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:2', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:2'),( 'be8ef4cb-ec72-4679-a488-5af1eb848d36:3', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:3', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:3', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:3', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:3', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:3', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:3', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:3', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:3', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:3', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:3', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:3', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:3', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:3', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:3', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:3', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:3', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:3', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:3', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:3', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:3', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:3', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:3', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:3', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:3'),( 'be8ef4cb-ec72-4679-a488-5af1eb848d36:4', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:4', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:4', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:4', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:4', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:4', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:4', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:4', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:4', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:4', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:4', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:4', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:4', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:4', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:4', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:4', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:4', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:4', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:4', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:4', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:4', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:4', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:4', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:4', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:4'),( 'be8ef4cb-ec72-4679-a488-5af1eb848d36:5', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:5', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:5', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:5', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:5', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:5', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:5', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:5', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:5', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:5', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:5', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:5', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:5', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:5', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:5', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:5', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:5', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:5', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:5', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:5', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:5', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:5', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:5', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:5', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:5'),( 'be8ef4cb-ec72-4679-a488-5af1eb848d36:6', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:6', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:6', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:6', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:6', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:6', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:6', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:6', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:6', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:6', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:6', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:6', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:6', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:6', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:6', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:6', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:6', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:6', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:6', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:6', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:6', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:6', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:6', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:6', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:6'),( 'be8ef4cb-ec72-4679-a488-5af1eb848d36:7', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:7', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:7', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:7', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:7', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:7', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:7', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:7', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:7', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:7', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:7', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:7', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:7', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:7', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:7', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:7', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:7', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:7', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:7', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:7', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:7', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:7', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:7', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:7', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:7'),( 'be8ef4cb-ec72-4679-a488-5af1eb848d36:8', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:8', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:8', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:8', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:8', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:8', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:8', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:8', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:8', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:8', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:8', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:8', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:8', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:8', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:8', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:8', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:8', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:8', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:8', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:8', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:8', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:8', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:8', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:8', 'be8ef4cb-ec72-4679-a488-5af1eb848d36:val:8') was aborted: ERROR: [Not found (yb/master/catalog_manager.cc:11692): LookupByIdRpc(tablet: 0fed15a16c2d4f70b4bcd5cd52c0852f, num_attempts: 1) failed: Tablet deleted: Table deleted at 2023-08-29 09:11:05 UTC (split child tablet IDs []), Not found (yb/master/catalog_manager.cc:11692): LookupByIdRpc(tablet: 8bba9dd9308e471fbcdb90d096ee2411, num_attempts: 1) failed: Tablet deleted: Table deleted at 2023-08-29 09:11:05 UTC (split child tablet IDs []), Not found (yb/master/catalog_manager.cc:11692): LookupByIdRpc(tablet: e8eda1400e524da3952d81a4290107c8, num_attempts: 1) failed: Tablet deleted: Table deleted at 2023-08-29 09:11:05 UTC (split child tablet IDs [])]  Call getNextException to see other errors in the batch.
`

**Comment by arybochkin** [comment]
From the master leader:
```
I0829 09:09:08.765424 44906 catalog_manager.cc:3637] CreateTable from 172.151.47.184:41787:
name: "tbl_no_explicit_lock"
table_id: "000033f3000030008000000000004014"
...
I0829 09:11:05.602205 44912 catalog_manager.cc:5919] Servicing DeleteTable request from 172.151.23.235:43497: table { table_id: "000033f3000030008000000000004014" } is_index_table: false
...
I0829 09:11:05.665913 44888 catalog_manager.cc:6496] Servicing IsDeleteTableDone request for table id 000033f3000030008000000000004014: totally deleted
...
I0829 09:39:36.381310 44888 catalog_manager.cc:3637] CreateTable from 172.151.23.235:44801:
name: "tbl_no_explicit_lock"
table_id: "000033f3000030008000000000004014"
```

The same OIDs range is shared across different nodes. The fix is being prepared in the context of #16130
